<?xml version="1.0"?>
<launch>
  <!-- declare arg to be passed in -->
  <arg name="team_side" default="home"/>

  <!-- declare machines -->
  <machine name="server_home" address="192.168.1.186" user="ecestudent" env-loader="~/env.sh"/>
  <machine name="server_away" address="192.168.1.160" user="ecestudent" env-loader="~/env.sh"/>

  <!-- run on machine -->
  <node machine="server_$(arg team_side)" name="usb_cam_$(arg team_side)" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video0"/>
    <param name="image_width" value="864"/>
    <param name="image_height" value="480"/>
    <param name="pixel_format" value="yuyv"/>
    <param name="camera_frame_id" value="usb_cam"/>
    <param name="io_method" value="mmap"/>
  </node>
</launch>